dawn/examples/python/test_python.cmake

```
@@ -0,0 +1,34 @@
+cmake_minimum_required(VERSION 3.12.4)
+if(NOT DEFINED PYTHON_EXECUTABLE)
+    find_package(PythonInterp 3 REQUIRED)
```

This was deprecated. https://cmake.org/cmake/help/git-stage/module/FindPythonInterp.html?highlight=pythoninterp
```suggestion
    find_package(Python3 COMPONENTS Interpreter REQUIRED)
```